pkgs/development/python-modules/streamingjson/default.nix

```
@@ -0,0 +1,35 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchPypi,
+  pythonOlder,
+  setuptools,
+}:
+
+buildPythonPackage rec {
+  pname = "streamingjson";
+  version = "0.0.5";
+  pyproject = true;
+
+  disabled = pythonOlder "3.7";
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-k7AofOStQIkwj75XRbi/1js55EWm84kUdOqEkShKI+M=";
+  };
+
+  build-system = [ setuptools ];
+
+  dependencies = [ ];
+
+  doCheck = false;
+
+  pythonImportsCheck = [ "streamingjson" ];
+
+  meta = with lib; {
+    description = "Streamlined, user-friendly JSON streaming preprocessor";
+    homepage = "https://pypi.org/project/streamingjson/";
+    license = licenses.mit;
+    maintainers = with maintainers; [ ethanthoma ];
+  };
```

`with` before attrsets (especially `with lib;`, _especially_ file-wide ones, _**especially**_ multiple ones) has been abused to death in the past and is now frowned upon[^1]. Just prepend `lib.` where necessary

[^1]: `with` before small scopes, like lists, is fine

```suggestion
  meta = {
    description = "Streamlined, user-friendly JSON streaming preprocessor";
    homepage = "https://pypi.org/project/streamingjson/";
    license = lib.licenses.mit;
    maintainers = with lib.maintainers; [ ethanthoma ];
  };
```

---

no need to reply in each convo, just mark as resolved when it's resolved